engines/wintermute/utils/path_util.cpp

```diff
@@ -74,6 +74,11 @@ AnsiString PathUtil::getDirectoryName(const AnsiString &path) {
 AnsiString PathUtil::getFileName(const AnsiString &path) {
 	AnsiString newPath = unifySeparators(path);
 	Common::String lastPart = Common::lastPathComponent(newPath, '/');
+
+	if (lastPart.empty()) {
+		return path;
+	}
```

Comment:
Technically, the following code should be else-if to this I guess.

On the other hand, does lastPathComponent actually return strings with the separator still in them? (Or could we simply do else { return lastPart; } ?